tests/system/provisioning/manager_agent/vars/configurations.yml

```diff
@@ -3,25 +3,25 @@ cluster_key: "00000000000000000000000000000000"
 manager_hostname: "wazuh-manager"
 
 log_file: "/var/log/secure" #Remember to change in agents ossec.conf if changed
-agent_user: "ossec" #Only valid for agents verions<4.2.5
+
 restart_command_415: /var/ossec/bin/ossec-control restart
 restart_command_4_2: /var/ossec/bin/wazuh-control restart
 
 agent1_id: "001"
 agent1_hostname: "wazuh-agent1"
 agent1_key: "1111111111111111111111111111111111111111111111111111111111111111"
 agent1_package: "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_4.1.5-1_amd64.deb"
+agent1_user: "ossec" #Only valid for agents verions<4.2.5
 
 agent2_id: "002"
 agent2_hostname: "wazuh-agent2"
 agent2_key: "2222222222222222222222222222222222222222222222222222222222222222"
 agent2_package: "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_4.2.2-1_amd64.deb"
-agent_user_4_2: "wazuh"
-
 agent3_id: "003"
 agent3_hostname: "wazuh-agent3"
 agent3_key: "3333333333333333333333333333333333333333333333333333333333333333"
-agent3_package: "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_4.2.5-1_amd64.deb"
+agent3_package: "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_{{package_version}}-{{package_revision}}_amd64.deb"
```

Comment:
The problem with this structure is that most of system test are executed using packages from packages-dev.com/pre-elease.

Example:
If ansible is executed using:

```
ansible-playbook -i inventory.yml playbook.yml --extra-vars='{"package_repository":"packages-dev", "repository": "pre-release", "package_version": "4.8.0", "package_revision": "1"}'
```
The provision of agent-3 will fail because the package in the URL will not exist:


```
TASK [roles/agent-role : Create log source] *****************************************************************************************************************************
changed: [wazuh-agent3]

TASK [roles/agent-role : Download package] ******************************************************************************************************************************
fatal: [wazuh-agent3]: FAILED! => changed=false 
  dest: /tmp/wazuh-agent.deb
  elapsed: 0
  msg: Request failed
  response: 'HTTP Error 404: Not Found'
  status_code: 404
  url: https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent_4.8.0-1_amd64.deb

PLAY RECAP **************************************************************************************************************************************************************
localhost                  : ok=9    changed=5    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
wazuh-agent1               : ok=12   changed=11   unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
wazuh-agent2               : ok=12   changed=11   unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
wazuh-agent3               : ok=4    changed=3    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
wazuh-manager              : ok=13   changed=12   unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4808/commits/9c8de595af24e1317cef5a94f90e8c6f58519461